src/qt/bitcoin.cpp

```diff
@@ -535,13 +536,25 @@ int main(int argc, char *argv[])
 #ifdef Q_OS_MAC
     QApplication::setAttribute(Qt::AA_DontShowIconsInMenus);
 #endif
+    {
+        QSslConfiguration sslconf = QSslConfiguration::defaultConfiguration();
 #if QT_VERSION >= 0x050500
-    // Because of the POODLE attack it is recommended to disable SSLv3 (https://disablessl3.com/),
-    // so set SSL protocols to TLS1.0+.
-    QSslConfiguration sslconf = QSslConfiguration::defaultConfiguration();
-    sslconf.setProtocol(QSsl::TlsV1_0OrLater);
-    QSslConfiguration::setDefaultConfiguration(sslconf);
+        // Because of the POODLE attack it is recommended to disable SSLv3 (https://disablessl3.com/),
+        // so set SSL protocols to TLS1.0+.
+        sslconf.setProtocol(QSsl::TlsV1_0OrLater);
+#else
+        // QT versions prior to 5.5 do not have the flag for saying "TLSv1 or later" (https://bugreports.qt.io/browse/QTBUG-43168)
+        // We don't have enum values for 1.1 or 1.2, nor for whatever may come next, so we must manually go through the list and strip anything from SSLv3 or earlier
```

Comment:
AFAIK this doesn't work: disabling pre-TLSv1 ciphers is orthogonal to disabling the pre-TLSv1 protocols. The ciphers are tagged with the protocol version in which they were introduced, but it isn't the ciphers that are insecure but the protocol itself.
So from what I understand this will reduce the choices of cipher arbitrarily to recent ones, possibly causing incompatibilities, but will not prevent SSLv3 related attacks.